package.json

```diff
@@ -24,7 +24,7 @@
     "test:watch": "yarn test --watch",
     "prebuild": "yarn test --ci && shx rm -rf dist",
     "build": "yarn build:js && yarn build:sass",
-    "build:js": "babel src --config-file ./config/babel.config.js --out-dir dist/js --only 'src/components/**/*.js,src/index.js' --ignore '**/tests/**,**/fixtures/**'",
+    "build:js": "babel src --config-file ./config/babel.config.js --out-dir dist/js --only 'src/components/**/*.js,src/constants/**/*.js,src/k8s/**/*.js,src/models/**/*.js,src/utils/**/*.js,src/index.js' --ignore '**/tests/**,**/fixtures/**'",
```

Comment:
can't we simplify that to just `src/**/*.js` ?

Reply:
added `src/**/*.js` and ignore now includes `jest` and `cosmos` folders as there releate to tests which we are not shipping